### DISCUSSION TITLE
Change height of navigation tabs

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [squidfunk/mkdocs-material#4563] [Q&A]
Is it possible to change the height of the navigation tabs ?

**Comment by squidfunk** [comment]
Yes, [use additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css).

**Comment by SoumayaMauthoorMOJ** [comment]
Sorry I should have asked - how do I change the height of the navigation tabs ?
